### PR TITLE
feat(hippy-vue-next): support vue3.x

### PR DESCRIPTION
**Pull request by lxy32167**
# Pre-PR Checklist

- [x] I added/updated relevant documentation.
- [x] I followed the [Convention Commit](https://conventionalcommits.org/) guideline with maximum 72 characters to submit commit message.
- [x] I squashed the repeated code commits.
- [x] I signed the [CLA].
- [x] I added/updated test cases to check the change I am making.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2357) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/Hippy?pullRequest=2357) before we can accept your contribution.<br/><hr/>**mitnickliu** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/Hippy?pullRequest=2357) it.</sub>

File: scripts/build-example.js [hunk 1]
```diff
@@ -51,7 +51,10 @@ console.log(`1/3 Start to install ${example} dependencies.`);
 exec('npm install --legacy-peer-deps', execOptions);
 
 console.log(`2/3 Start to build project ${example}.`);
-exec('npm run hippy:vendor', execOptions); // Build vendor js
+if (example !== 'hippy-vue-next-demo') {
```
[the remark this answers is not in the document]
Comment (gguoyu) [reply scripts/build-example.js:54]: 这里考虑到vendor会被内置，改动起来需要跟随版本，如果有bug不易处理。所以想等跑一段时间稳定后，再build vendor做内置

**Review by gguoyu — COMMENTED** [review]
回复问题

File: packages/hippy-vue-next-style-parser/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@hippy/hippy-vue-next-style-parser",
+  "version": "2.2.1",
```
Comment (zoomchan-cxj) [inline packages/hippy-vue-next-style-parser/package.json:3]: 统一改成 unspecified

File: packages/hippy-vue-next/package.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@hippy/vue-next",
+  "version": "2.2.1",
```
Comment (zoomchan-cxj) [inline packages/hippy-vue-next/package.json:3]: 统一改成 unspecified

**Review by zoomchan-cxj — APPROVED** [review]
LGTM

**Pull request merged**